Workers/DatabaseSyncWorker/DatabaseSyncLogic.cs

```diff
@@ -145,22 +194,12 @@ public void ProcessOpList(OpList opList)
                     case DatabaseSyncService.Commands.Delete:
                         HandleDeleteRequest(commandRequestOp);
 
-                        break;
-                    case DatabaseSyncService.Commands.GetMetrics:
-                        DatabaseSyncService.SendGetMetricsResponse(connection, commandRequestOp.RequestId, new GetMetricsResponse(ImmutableDictionary<uint, long>.Empty, Improbable.Postgres.Metrics.GetCounts(), Improbable.Postgres.Metrics.GetTimingStats()));
-
-                        break;
-                    case DatabaseSyncService.Commands.AssociatePathWithClient:
-                        HandleAssociatePathWithClient(commandRequestOp);
-
                         break;
 
                     case DatabaseSyncService.Commands.Batch:
                         HandleBatch(commandRequestOp);
 
                         break;
-                    default:
```

Comment:
What's the rationale for removing the `default` branch? Do we expect to handle any other commands elsewhere?

Reply:
The intent was logging an error rather than blowing up with an exception - not sure what happened there.